crates/nu-command/src/filters/default.rs

```diff
@@ -27,7 +27,7 @@ impl Command for Default {
     }
 
     fn usage(&self) -> &str {
-        "Sets a default row's column if missing."
+        "Sets a default value to use if a row's column is missing or null."
```

Comment:
I think the "to use" makes it less clear
```suggestion
        "Sets a default value if a row's column is missing or null."
```